docs/install.md

```diff
@@ -42,6 +47,21 @@ Compose can also be installed as a Python package:
 
     $ sudo pip install -U docker-compose
 
+### Container install
+
+Compose can also be run inside a container, from a small bash script wrapper.
+To install compose as a container run:
+
+    curl -L https://github.com/docker/compose/releases/download/1.3.3/compose-run > /usr/local/bin/docker-compose
```

Comment:
Ya, since this command is really just printing out the bash script, we could probably call it `echo-run-script` or something like that?